packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -82,7 +85,13 @@ export class TdsTooltip {
           }}
           selector={this.selector}
           referenceEl={this.referenceEl}
-          trigger={this.mouseOverTooltip ? 'hover-popover' : 'hover'}
+          trigger={
```

Comment:
Let's try and avoid double turnaries

Reply:
Agree! Maybe a method for handling it instead? Something like:

```
determineTrigger() {
    if (this.trigger === 'hover') {
      return this.mouseOverTooltip ? 'hover-popover' : 'hover';
    }
    return this.trigger;
  }
```
And then use it:
`trigger={this.determineTrigger()}`